sanic/request.py

```diff
@@ -274,6 +306,12 @@ def remote_addr(self):
 
     @property
     def scheme(self):
+        forwarded_proto = self.headers.get(
+            "x-forwarded-proto"
+        ) or self.headers.get("x-scheme")
```

Comment:
Isn't this header `X-Url-Scheme` and not `X-Scheme` ?

Reply:
I'll add a full comment regarding this.